spec/models/tag_spec.rb

```diff
@@ -96,11 +96,11 @@
 
   describe 'taggings scopes' do
     let(:first) do
-      described_class.create!(name: Faker::Science.science, creator: create(:user), status: 'approved')
+      described_class.create!(name: Faker::Science.unique.science, creator: create(:user), status: 'approved')
```

Comment:
This is unrelated to PR, but I faced spec error. It seems, that sometimes Faker produced two tags with same name, and it caused spec to fail. Added unique instruction, to make Faker to generate unique values.